src/test/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImplTest.java

```diff
@@ -350,6 +356,20 @@ private DocumentService buildMockDocumentService() throws Exception {
         EasyMock.replay(documentService);
         return documentService;
     }
+    
+    private ParameterService buildParameterervice() {
+        ParameterService parameterService = EasyMock.createMock(ParameterService.class);
+        
+        EasyMock.expect(
+                parameterService.getParameterValueAsString(KFSConstants.ParameterNamespaces.FINANCIAL, 
+                CuFPParameterConstants.CreateAccountingDocumentService.CREATE_ACCOUNTING_DOCUMENT_SERVICE_COMPONENT_NAME, 
+                CuFPParameterConstants.CreateAccountingDocumentService.CREATE_ACCT_DOC_REPORT_EMAIL_ADDRESS))
+        .andStubAnswer(() -> "kfs-gl@cornell.edu");

```

Comment:
Should we move "kfs-gl@cornell.edu" to a constant or property or is it okay because it's within a test class? Should the email address match what is in the SQL script (kfs-gl_fp@cornell.edu)?

Reply:
I think the in-line String is OK here because the value isn't validated.  The service call does need to return a value, so that is why I did the Mock.  However, to avoid confusion, I shall change this to be what is in the SQL.